bigquery/samples/client_query_batch.py

```diff
@@ -32,13 +32,11 @@ def client_query_batch(client):
     """
 
     # Start the query, passing in the extra configuration.
-    query_job = client.query(
-        sql, location="US", job_config=job_config
-    )  # Make an API request.
+    query_job = client.query(sql, job_config=job_config)  # Make an API request.
 
     # Check on the progress by getting the job's updated state. Once the state
     # is `DONE`, the results are ready.
-    query_job = client.get_job(query_job.job_id, location="US")  # Make an API request.
+    query_job = client.get_job(query_job.job_id)  # Make an API request.
```

Comment:
Actually, I don't believe `get_job` does any auto-detection except for defaulting to `US`. Luckily, we can extract the location from the job resource:

```suggestion
    query_job = client.get_job(query_job.job_id, location=query_job.location)  # Make an API request.
```